cmake/Config.cmake

```diff
@@ -19,8 +19,21 @@ if (WIN32)
             "${CMAKE_CURRENT_SOURCE_DIR}/cs/test/compat/bin/retail")
 endif()
 
-find_package (PythonLibs 2.7)
-find_package (PythonInterp 2.7)
+# set python version
+if (NOT DEFINED BOND_PYTHON_VERSION)
```

Comment:
`FindPython.cmake` defines `Python_ADDITIONAL_VERSIONS` variable. Could we use it to control user preferred version instead of defining a Bond specific variable?